clijs/test/globalSetup.ts

```diff
@@ -0,0 +1,46 @@
+import { type ChildProcess, spawn } from "node:child_process";
+import { closeSync, openSync } from "node:fs";
+import { testEnv } from "./testEnv.js";
+
+interface Nild {
+  pid: number;
+  process: ChildProcess;
+  logFd: number;
+}
+
+let nildInstance: Nild;
+
+export async function setup() {
+  console.log("launching nild:", testEnv.nild);
+  const logFd = openSync("nild.log", "w");
```

Comment:
I don't see any logs in CI console. Will be it possible to see a content of that file in case of test failure?

Reply:
are you sure you need the whole `nild.log` in CI console? I suggest keeping only test logs there. the rest can be inspected during local run (btw currently it works in the same way)

Reply:
It would we useful in case of failure. Sometimes some issues happen only in CI.

Reply:
ok, changed this. if it turns out to be inconvenient, we can always turn off